widget/embedded/src/hooks/useConfirmSwap.ts

```diff
@@ -1,53 +1,49 @@
 /* eslint-disable @typescript-eslint/no-magic-numbers */
 import type {
```

Comment:
Maybe it's not in the scope of this task but could you use `usePrepareBlockchainList` structure for this hook as well? I mean move helpers to a separate file. then we can think about how we can write tests.